src/cow-react/modules/swap/pure/EthFlow/EthFlowStepper/StatusIcon.tsx

```diff
@@ -3,7 +3,7 @@ import styled from 'styled-components/macro'
 import { transparentize } from 'polished'
 import SVG from 'react-inlinesvg'
 
-export type StatusIconState = 'success' | 'pending' | 'not-started' | 'error'
+export type StatusIconState = 'success' | 'pending' | 'not-started' | 'error' | 'skipped'
```

Comment:
why skipped for "cancelled"? Is not canceled good?

Reply:
Good point, will update